spec-tests/src/test/java/org/candlepin/spec/owners/OwnerResourceSpecTest.java

```diff
@@ -792,4 +795,41 @@ public void shouldAllowUserWithOwnerPoolsPermissionCanSeeSystemPurposeOfTheOwner
         assertEquals(owner.getKey(), attributes.getOwner().getKey());
     }
 
+    @Test
+    @OnlyInHosted
+    public void shouldMigrateAllConsumersOfAnonymousOwnerToTargetOwner() {
+        String accountId = StringUtil.random("cloud-account-id-");
+        String instanceId = StringUtil.random("cloud-instance-id-");
+        String offerId = StringUtil.random("cloud-offer-");
+        OwnerDTO anonOwner = admin.owners().createOwner(Owners.random().anonymous(true));
+        OwnerDTO destOwner = owners.createOwner(Owners.random());
+        admin.hosted().createOwner(anonOwner);
+        ProductDTO prod = admin.ownerProducts().createProduct(anonOwner.getKey(), Products.random());
+        admin.hosted().createProduct(prod);
+        admin.owners().createPool(anonOwner.getKey(), Pools.random(prod));
```

Comment:
same here: use the existing `owners` field, or remove it from the setup method and use `admin.owners()` everywhere

Reply:
Done.